19.0 t 56583 jjs 24 12

jjscarafia opened the pull request.

jjscarafia committed:
[FIX] _reports: quincenal returns filter on reports

arreglar cambiar fecha en reportes si return type quincenal

jjscarafia committed:
[IMP] _reports: add partners and branch allowed

jjscarafia committed:
[IMP] user fortnightly for sicore

jjscarafia committed:
[ADD] account_reports: auto generate tax returns

jjscarafia committed:
[IMP] _account_reports: do not set deadline_days_delay

(keep as in odoo data, only set default_deadline_days_delay)

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/855.png)](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/855)

jjscarafia committed:
[IMP] _reports: minor fixes

estabamos haciendo overide de un metodo solo para sacar _ensure_tax_group_configuration_for_tax_closing

no debreía ser necesario, eventualmente tendremos que heredar "_ensure_tax_group_configuration_for_tax_closing"

jjscarafia commented:
@roboadhoc r+ rebase-ff

roboadhoc commented:
Merge method set to rebase and fast-forward.

jjscarafia commented:
@roboadhoc r-

jjscarafia committed:
[FIX] _reports: domains

jjscarafia commented:
@roboadhoc r+ rebase-ff

roboadhoc commented:
Merge method set to rebase and fast-forward.
